ci/README.md

```diff
@@ -8,6 +8,13 @@ image at [`lyft/envoy:<hash>`](https://hub.docker.com/r/lyft/envoy/) is an image
 corresponds to the master commit at which the binary was compiled. Lastly, `lyft/envoy:latest` contains
 an Envoy binary built from the latest tip of master that passed tests.
 
+# Using alpine envoy image
```

Comment:
Please mention as above that both this images have a `hash` tag and a `latest` tag.